src/reporter.test.ts

```diff
@@ -318,7 +318,7 @@ describe('reporter', () => {
               },
               response: {
                 truncated: true,
-                body: `{"a":"${value}","b":"${value}","key":"****","password":"****","e":"${value}","secret":"****","f":"${value}","g":"${value}"}...[too long]`,
+                body: `{"a":"${value}","b":"${value}","key":"****","password":"****","e":"${value}","secret":"****","f":"${value}","g":"${value}"}✂`,
```

Comment:
i'm not sure how i feel about this, because it is cute - aside from it being less clear and difficult to identify, are we sure there's no chance of it causing problems in the frontend?

Reply:
It has been tested and it's already shipped with the OpenTelemetry JS distro